opencog/nlp/relex2logic/post-processing.scm

```diff
@@ -252,7 +252,7 @@
 		(list
 			(ForAllLink
 				(VariableNode "$X")
-				(ImplicationLink
+				(ImplicationScopeLink
```

Comment:
this does not look right -- I think the variable $X is supposed to be that in the ForAllLink -- this implication link is assumed to be not scoped.